providers/common/fipsindicator.c

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Should this be conditional on FIPS_MODULE?
If so, it shouldn't be in `providers/common`.

Reply:
Moved, the header is however common.